symfony

```diff
@@ -0,0 +1,188 @@
+#!/bin/sh
```

Comment:
It is based on the magento executable in the [dockerized magento](https://github.com/andreaskoch/dockerized-magento) stack. So it's edited